CONTRIBUTING.md

```diff
@@ -23,3 +23,34 @@ use GitHub pull requests for this purpose. Consult [GitHub Help] for more
 information on using pull requests.
 
 [GitHub Help]: https://help.github.com/articles/about-pull-requests/
+
+## Building Dagger
+
+Dagger is built with [`bazel`](https://bazel.build).
+
+### Building Dagger from the command line
+
+* Go to the [Bazel installation page](https://docs.bazel.build/versions/master/install.html) and install Bazel.
+  * Learn more about Bazel targets [here](https://docs.bazel.build/versions/master/build-ref.html).
+* Clone the Dagger repository: `git clone git@github.com:google/dagger.git dagger`
+* Build the Dagger project with `bazel build`
```

Comment:
Note this doesn't do anything, I think it needs to be `bazel build //...`?

```
❯ bazel build
Starting local Bazel server and connecting to it...
INFO: Invocation ID: 02428c04-950b-4954-8c8e-eec1014b5457
INFO: SHA256 (https://github.com/google/bazel-common/archive/1c225e62390566a9e88916471948ddd56e5f111c.zip) = 85e2b5ee70ca5616f1cd43b6b7ee080050135b4300fb1027acbcd98de7fb3df0
INFO: Analysed 0 targets (0 packages loaded, 0 targets configured).
INFO: Found 0 targets...
INFO: Elapsed time: 2.842s, Critical Path: 0.02s
INFO: 0 processes.
INFO: Build completed successfully, 1 total action
```

Reply:
I'm getting a bunch of problems when building 🤔 
```bazel
ERROR: /private/var/tmp/_bazel_zhukov/ec789c8512b32a6fa14b8de7a8b5d17e/external/com_google_protobuf/BUILD:597:1: Traceback (most recent call last):
	File "/private/var/tmp/_bazel_zhukov/ec789c8512b32a6fa14b8de7a8b5d17e/external/com_google_protobuf/BUILD", line 597
		internal_gen_well_known_protos_java(srcs = WELL_KNOWN_PROTOS)
	File "/private/var/tmp/_bazel_zhukov/ec789c8512b32a6fa14b8de7a8b5d17e/external/com_google_protobuf/protobuf.bzl", line 266, in internal_gen_well_known_protos_java
		Label(("%s//protobuf_java" % REPOSITOR...))
	File "/private/var/tmp/_bazel_zhukov/ec789c8512b32a6fa14b8de7a8b5d17e/external/com_google_protobuf/protobuf.bzl", line 266, in Label
		REPOSITORY_NAME
builtin variable 'REPOSITORY_NAME' is referenced before assignment.
ERROR: /private/var/tmp/_bazel_zhukov/ec789c8512b32a6fa14b8de7a8b5d17e/external/com_google_protobuf/BUILD:380:1: Target '@com_google_protobuf//:android' contains an error and its package is in error and referenced by '@com_google_protobuf//:protoc'
ERROR: /private/var/tmp/_bazel_zhukov/ec789c8512b32a6fa14b8de7a8b5d17e/external/com_google_protobuf/BUILD:380:1: Target '@com_google_protobuf//:windows' contains an error and its package is in error and referenced by '@com_google_protobuf//:protoc'
ERROR: /private/var/tmp/_bazel_zhukov/ec789c8512b32a6fa14b8de7a8b5d17e/external/com_google_protobuf/BUILD:380:1: Target '@com_google_protobuf//:windows_msvc' contains an error and its package is in error and referenced by '@com_google_protobuf//:protoc'
ERROR: /Users/zhukov/Development/GitHub/google/dagger/javatests/dagger/grpc/functional/server/BUILD:16:1: every rule of type proto_library implicitly depends upon the target '@com_google_protobuf//:protoc', but this target could not be found because of: Target '@com_google_protobuf//:protoc' contains an error and its package is in error
ERROR: Analysis of target '//javatests/dagger/grpc/functional/server:coffee_service_java_proto' failed; build aborted: Analysis failed
INFO: Elapsed time: 0.475s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (0 packages loaded, 0 targets configured)
```

Reply:
Perhaps `bazel build <target>` or `bazel build java/...`